### ISSUE TITLE
Release of next version

### ISSUE THREAD

**Issue by observingClouds** [zarr-developers/numcodecs#324]
Hi everyone,

I was wondering if there is a rough timeline for the next release of numcodecs. I think there is currently a lot of interest in the new implementations of e.g. registering algorithms via entry points and the `bitrounding` algorithm. I was wondering if we could create a milestone or just list issues which should be solved before the next release, so we (the community) might help 😄 

On that note, it would be great if we could publish a new release also to pypi, so that the new version of numcodecs can also be used as a dependency in other packages that shall be published on pypi.

Thanks!

**Comment by jakirkham** [comment]
We made some pretty significant changes in Zarr as part of the v3 spec (see [blogpost]( https://zarr.dev/blog/pre-release-2-12/ )) and are trying to digest those (and see if anything pops up)

The last few Numcodecs release have been alphas. It could make sense to do another alpha. This might help us catch any issues with Numcodecs before doing a final release

@joshmoore wdyt?

**Comment by observingClouds** [comment]
Thanks @jakirkham for the status update.

**Comment by joshmoore** [comment]
I remember doing the first alpha in order to get some testing of Entrypoints (which @cgohlke kindly did) After that I assume it was the beginning of @madsbk's work that kept the alphas going. Now? I'm not recalling anything. So no objections from my side if someone would like to push forward with 0.10. 

```
434c7b1 (origin/main) Move `master` to `main` (#322) (2 weeks ago) <jakirkham>
aab5392 Bitround Codec (#299) (2 weeks ago) <Ryan Abernathey>
eacd6e2 Drop `fastparquet` benchmark (#321) (4 weeks ago) <jakirkham>
ab92198 Trim wheel builds (#320) (4 weeks ago) <jakirkham>
581848e Drop Python 3.6 (#318) (4 weeks ago) <Josh Moore>
15f1713 (tag: v0.10.0.a3, origin/master, origin/HEAD) Add CHANGELOG for prereleases (2 months ago) <jmoore>
bedb8b0 Support of alternative array classes: ndarray-like (#305) (3 months ago) <Mads R. B. Kristensen>
f4196a4 Introduce a flat option to ensure_contiguous_ndarray to switch off flatten for ZFPY codec (#307) (3 months ago) <haiying xu>
3229a74 Fix macOS Python 3.10 (#311) (3 months ago) <jakirkham>
fca7c73 (tag: v0.10.0.a2) chore: bump cibuildwheel version, use action (#309) (3 months ago) <Henry Schreiner>
0b54e9c Codex: make encode and decode @abstractmethods (#306) (4 months ago) <Mads R. B. Kristensen>
a81856e Bump numpy from 1.19.0 to 1.21.0 (#301) (4 months ago) <dependabot[bot]>
ed22a59 (tag: v0.10.0.a1) Entrypoints (#300) (4 months ago) <Martin Durant>
500c048 Unnecessary `None` provided to `get()` (#295) (6 months ago) <Dimitri Papadopoulos Orfanos>
4d70c85 Unused variable (#294) (6 months ago) <Dimitri Papadopoulos Orfanos>
16de784 Unnecessary comprehension (#293) (6 months ago) <Dimitri Papadopoulos Orfanos>
dff4885 Codespell (#292) (6 months ago) <Dimitri Papadopoulos Orfanos>
e364ceb Fix expected result test for Shuffle. (#282) (8 months ago) <Elliott Sales de Andrade>
```
